cfgov/v1/util/rich_text.py

```diff
@@ -0,0 +1,425 @@
+import re  # parsing HTML with regexes LIKE A BOSS.
```

Comment:
Where does this file come from? Are there any tests?

Reply:
This Wagtail file was removed in Wagtail 2. 
I was hoping to keep Hallo functionality alive until we jump to Draftail, and the Wagtail tests for it use a detailed fixture that doesn't load in our environment. 

I would favor jumping straight to Draftail with our initial Wag2 upgrade, and spend our time testing and configuring that rather than working on tests for Hallo's rich_text operations.

Reply:
> I would favor jumping straight to Draftail with our initial Wag2 upgrade, and spend our time testing and configuring that rather than working on tests for Hallo and rich_text.py.

:+1: to this. If that decision is uncontroversial, I will look into customizing Draftail in the same was as we've customized Hallo.

Reply:
Okay. I think we could probably jump straight to Draftail with this PR. Remembering that we're not looking for running-functional here, just testable (not even tests passing).